test/test_support.ts

```diff
@@ -173,6 +173,19 @@ export class GoldenFileTest {
         .map(f => path.join(this.path, GoldenFileTest.tsPathToJs(f)));
   }
 
+  /**
+   * Find the absolute path to the tsickle root directory by reading the
+   * symlink bazel puts into bazel-bin back into the test_files directory
+   * and chopping off the test_files portion.
+   */
+  get workspaceRoot(): string {
+    if (!this.tsFiles) {
+      return '';
+    }
+    const resolvedFileSymLink = fs.readlinkSync(path.join(this.path, this.tsFiles[0]));
+    return resolvedFileSymLink.replace(/\/test_files.*$/, '');
```

Comment:
this won't work on windows, and I think we still have some people interested in that.

Can you use path.separator here and split based on it, then drop segments after the first `test_file`? Maybe there's also a more elegant formulation.

Reply:
Done.